tests/js/objects/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -113,6 +113,32 @@ const a3 = {
 ================================================================================
 `;
 
+exports[`invalid-accessor-generator.js format 1`] = `
+====================================options=====================================
+parsers: ["babel"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+// Invalid, but recoverable
+// https://github.com/babel/babel/pull/11921/
+
+({
+    get *iterator() { },
+    set *iterator(iter) { }
+});
+
+=====================================output=====================================
+// Invalid, but recoverable
+// https://github.com/babel/babel/pull/11921/
+
+({
+  get iterator() {},
+  set iterator(iter) {},
```

Comment:
Should we throw a error? The `*` is lost, user might don't expect it.

Reply:
Not sure it is our part

Reply:
We can throw if we want.

Reply:
@JLHwung Sorry to bother, but is it possible to recover this error with `generator` set to true? It will be easier to handle.

Reply:
I don't have strong opinion here, but let's try to do it

Reply:
@fisker Sure. I didn't set `generator` as `true` because accessor can not be a generator. If it is convenient for prettier I can draft a patch.

Reply:
Thank you!

Reply:
Thank you